invenio_jsonschemas/errors.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Invenio-JSONSchemas errors."""
+
+from __future__ import absolute_import, print_function
+
+
+class JSONSchemaError(Exception):
+    """Base class for errors in Invenio-JSONSchemas module."""
+
+
+class JSONSchemaNotFound(Exception):
+    """Exception raised when a requested JSONSchema is not found."""
+
+    def __init__(self, schema, *args, **kwargs):
+        """Constructor.
+
+        :param schema: path of the requested schema which was not found.
+        """
+        self.schema = schema
+        self.message = 'Schema "{}" not found'.format(schema)
```

Comment:
```python
super(JSONSchemaNotFound, self).__init__('Schema "{}" not found'.format(schema), *args, **kwargs)
```

Otherwise ``*args`` (which is passed down to ``Exception``), will overwrite ``self.message``.